xbmc/video/VideoThumbLoader.cpp

```diff
@@ -436,7 +436,7 @@ bool CVideoThumbLoader::FillLibraryArt(CFileItem &item)
     m_videoDatabase->Open();
     if (m_videoDatabase->GetArtForItem(tag.m_iDbId, tag.m_type, artwork))
       SetArt(item, artwork);
-    else if (tag.m_type == MediaTypeArtist)
+    else if (tag.m_type == "actor" && !tag.m_artist.empty())
```

Comment:
My idea for `m_type` when I initially added it was that it would always hold the media type an item belonged to. So when you are in the `Movies` node it would always be `MediaTypeMovie` independent of whether you are in the `Titles` or the `Actors` node. But I didn't really put any restrictions into the code and then it was suddenly used to also store other stuff which is why there is no `MediaTypeActor` and I'm reluctant to add it because an actor is not a media type. `MediaTypeArtist` also wasn't meant to be used in the video library / database but was meant for the music library.